standalone-e2e-test/pom.xml

```diff
@@ -0,0 +1,68 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.example</groupId>
```

Comment:
Could be com.github.lammaer. Naming conventions: https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

Reply:
Fixed